docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -35,60 +35,75 @@ Required connection parameters include the following:
 Microsoft Entra ID was previously known as Azure Active Directory (Azure AD).
 </Note>
 
-## Usage
-
-This integration can only be used to create chatbots for Microsoft Teams via the [`CREATE CHATBOT`](/agents/chatbot) syntax. Currently, *it cannot be used as a data source for other workloads*.
+### How to set up Microsoft Teams app
 
 Follow the instructions given below to set up the Microsoft Teams app that will act as the chatbot:
 
-  1. Follow [this link](https://dev.botframework.com/bots/new) to the Microsoft Bot Framework portal and sign in with your Microsoft account.
-  2. Fill out the *Display name*, *Bot handle*, and, optionally, the *Long description*, but leave the *Messaging endpoint* field empty for now.
-  3. Set the *App type* to be `Multi Tenant` and click on *Create Microsoft App ID and password*. This will open a new tab with the Azure portal.
-  4. Click on *New registration* and fill out the *Name* and select the `Accounts in any organizational directory (Any Azure AD directory - Multitenant)` option under *Supported account types*, and click on *Register*. Record the *Application (client) ID* for later use.
-  5. Click on *Certificates & secrets* under *Manage*.
-  6. Click on *New client secret* and fill out the *Description* and select an appropriate *Expires* period, and click on *Add*.
-  7. Copy the client secret and save it in a secure location.
+<Steps>
+  <Step title="Sign in to Microsoft and fill in the bot data">
```

Comment:
I think we should call this the 'Microsoft Bot Framework' portal.
Maybe we can say something along the lines of 'Register a bot in the Microsoft Bot Framework portal'.